remotespark/magics/sparkmagicsbase.py

```diff
@@ -47,7 +47,7 @@ def _sqlquery(cell, samplemethod, maxrows, samplefraction):
 
     @staticmethod
     def print_endpoint_info(info_sessions):
-        sessions_info = ["        {}".format(i) for i in info_sessions]
+        sessions_info = ["        {}\n".format(i) for i in info_sessions]
```

Comment:
I splitted Spark UI url and driver log url into its own line. Adding a new line between each session make separation between each session clearer.